requirements.txt

```diff
@@ -1,8 +1,13 @@
-Flask-Cache
-Flask-WTF==0.11
+python==2.7.12
+python-dev==2.7
```

Comment:
We are using 2.7.9 in production, on a Jessie install.
Maybe `>=2.7.9,<=2.7.12`?

Where does the 2.7.12 come from?